pkg/capr/installer/installer.go

```diff
@@ -80,6 +80,28 @@ func LinuxInstallScript(ctx context.Context, token string, envVars []corev1.EnvV
 	if token != "" {
 		token = "CATTLE_ROLE_NONE=true\nCATTLE_TOKEN=\"" + token + "\""
 	}
+
+	// Merge the env vars with the AgentTLSModeStrict
+	found := false
+	for _, ev := range envVars {
+		if ev.Name == "STRICT_VERIFY" {
```

Comment:
`"STRICT_VERIFY"` could probably be a constant in `capr/common.go`.

Reply:
I didn't want to make it a constant on our side yet given it's used for the `cluster-agent` as well as `system-agent`. We can revisit this later though